.github/workflows/build-scan-push.yml

```diff
@@ -11,14 +11,12 @@ jobs:
     outputs:
       image: ${{ steps.set-var.outputs.image }}
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - id: set-var
         run: |
-          content=`cat ./image-matrix.json`
-          content="${content//'%'/'%25'}"
-          content="${content//$'\n'/'%0A'}"
-          content="${content//$'\r'/'%0D'}"
-          echo "::set-output name=image::$content"
+          echo 'image<<EOF' >> $GITHUB_OUTPUT
+          cat ./image-matrix.json >> $GITHUB_OUTPUT
+          echo 'EOF' >> $GITHUB_OUTPUT
```

Comment:
Took me a moment to understand the new syntax for [multiline strings](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings), but is much simpler!